lib/prawn/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Prawn
-  VERSION = '2.4.0'
+  VERSION = '2.5.0'
```

Comment:
I understand the sentiment but generally this is not done in a PR. I have not decided yet on the version that's going to be released.

Reply:
Right makes sense. I put my reasoning there. https://github.com/prawnpdf/prawn/issues/1195#issuecomment-1797836020